### ISSUE TITLE
[Bug]: Cannot Convert SuryaOCR Detection Model `EfficientViTForSemanticSegmentation` to Openvino

### ISSUE THREAD

**Issue by bibekyess** [openvinotoolkit/openvino#30356]
### OpenVINO Version

2025.1.0

### Operating System

Windows System

### Device used for inference

CPU

### Framework

PyTorch

### Model used

EfficientViTForSemanticSegmentation (https://huggingface.co/vikp/surya_det3/blob/main/config.json)

### Issue description

I am trying to convert the surya-ocr detection model ([https://huggingface.co/vikp/surya_det3](https://huggingface.co/vikp/surya_det3))) to OpenVINO but I am getting this error. How to resolve this issue?
If there is some tutorial or some reference to bypass this layer and fix this, feel free to let me know.
Thanks!

The following error occurs when executing `ov.convert_model` command.
```
OpConversionFailure: Check 'is_conversion_successful' failed at src\frontends\pytorch\src\frontend.cpp:174:
FrontEnd API failed with OpConversionFailure:
Model wasn't fully converted.
Summary:
-- No conversion rule found for operations: aten::special_expit
```

### Step-by-step reproduction

```
from PIL import Image
from surya.detection import DetectionPredictor
import torch

det_predictor = DetectionPredictor()
model = det_predictor.model

class ModelWrapper(torch.nn.Module):
    def __init__(self, model):
        super().__init__()
        model = model.to('cpu')
        model = model.eval()
        self.model = model
    
    def forward(self, x):
        outputs = self.model(x)
        return outputs.logits

wrapped_model = ModelWrapper(model)
import openvino as ov

dummy_input = torch.randn(1, 3, 1200, 1200) 
ov_model = ov.convert_model(wrapped_model, example_input=dummy_input)

```

### Relevant log output

```shell

```

### Issue submission checklist

- [x] I'm reporting an issue. It's not a question.
- [x] I checked the problem with the documentation, FAQ, open issues, Stack Overflow, etc., and have not found a solution.
- [x] There is reproducer code and related data files such as images, videos, models, etc.

**Comment by bibekyess** [comment]
Hi @mvafin 
I was able to find a workaround by replacing `torch.special.expit(logits)` with `torch.sigmoid(logits)`. But I faced a new issue.
Somehow the model is running fine when offloading in CPU, but it gives totally junk results when offloading in GPU. 
The following is the code to reproduce and the models are attached. Feel free to check it. Thanks!

Output with TORCH CPU:
```
SemanticSegmenterOutput(loss=None, logits=tensor([[[[3.8690e-06, 1.1998e-06, 1.1137e-06,  ..., 9.4077e-07,
           6.3582e-07, 2.0356e-06],
          [1.2587e-06, 4.2859e-07, 5.0863e-07,  ..., 5.4160e-07,
           5.1652e-07, 2.1255e-06],
          [6.1256e-07, 1.4877e-07, 1.6849e-07,  ..., 1.6676e-07,
           2.0925e-07, 1.6451e-06],
          ...,
          [-0.9751, -0.8233, -1.8825,  ..., -0.6540, -0.2099, -0.3515],
          [-1.6355, -0.8812, -2.3937,  ..., -1.0807, -0.7290, -0.1428],
          [-2.3214, -1.0522, -3.0711,  ..., -0.7509, -0.9480, -0.8032]]]],
       grad_fn=<AddBackward0>)], attentions=None)
```

Output with OpenVINO CPU
```
{<ConstOutput: names[] shape[?,2,1..,1..] type: f32>: array([[[[3.91944195e-06, 1.21675976e-06, 1.13352189e-06, ...,
          9.56805366e-07, 6.44027637e-07, 2.06021923e-06],
         [1.27523469e-06, 4.34865882e-07, 5.17948934e-07, ...,
          5.50880145e-07, 5.23265840e-07, 2.14993702e-06],
         [6.19074285e-07, 1.50381609e-07, 1.70782613e-07, ...,
          1.68424862e-07, 2.10763872e-07, 1.65507481e-06],
         ...,
         [1.34800800e-07, 2.56258915e-07, 1.32642199e-07, ...,
          9.16313365e-06, 3.47041132e-05, 3.43794500e-06],
         [2.03017166e-07, 3.50684900e-07, 4.55876375e-07, ...,
          2.36571159e-05, 6.14023593e-05, 1.08614613e-05],
         [1.56802670e-07, 4.83635851e-08, 5.51237065e-08, ...,
          3.35114214e-06, 1.12983635e-05, 4.92054278e-06]]]],
      shape=(1, 2, 300, 300), dtype=float32)}
```

Output with OpenVINO GPU:
```
{<ConstOutput: names[] shape[?,2,1..,1..] type: f32>: array([[[[0.4885254 , 0.4885254 , 0.4885254 , ..., 0.4885254 ,
          0.4885254 , 0.4885254 ],
         [0.4885254 , 0.4885254 , 0.4885254 , ..., 0.4885254 ,
          0.4885254 , 0.4885254 ],
         [0.4885254 , 0.4885254 , 0.4885254 , ..., 0.4885254 ,
          0.4885254 , 0.4885254 ],
         ...,
         [0.48950195, 0.48950195, 0.48950195, ..., 0.48950195,
          0.48950195, 0.48950195]]]],
      shape=(1, 2, 300, 300), dtype=float32)}
```
Model: [surya_det3_ov](https://drive.google.com/file/d/1QeYWhQq3V3KBSCitwffVU1Vnc_irD0TN/view?usp=sharing)

Code:
```
from PIL import Image
from surya.detection import DetectionPredictor
import torch

IMAGE_PATH = "./image (11).png"
det_predictor = DetectionPredictor()

image = Image.open(IMAGE_PATH)
def build_example_input(image):
    input_values = det_predictor.prepare_image(image.convert("RGB"))

    return {"pixel_values": torch.unsqueeze(input_values, 0)}

example_input = build_example_input(image)
print(det_predictor.model(**example_input))

---

import openvino as ov
import torch

core = ov.Core()

# AUTO loads in CPU not in GPU, Directly specifying GPU runs without errors but bad result
compiled_model = core.compile_model("surya_det3_ov\surya_det3_ov.xml", "GPU")

print(compiled_model(example_input['pixel_values']))


```

**Comment by rkazants** [comment]
Hi @bibekyess, what torch version are you using? With latest torch version, TorchScript traces a model with `aten::sigmoid` operation and no `aten::special_expit`.

@Lyamin-Roman, could you please see inference issue on GPU?

Best regards,
Roman

**Comment by rkazants** [comment]
Support for `aten::special_expit`: https://github.com/openvinotoolkit/openvino/pull/30410

**Comment by bibekyess** [comment]
Hi @rkazants! Thank you for your response. I checked and I am using torch version: `Version: 2.7.0+cpu` and `python==3.11.9`.
```
Device Details
Device Name: LG Gram Pro Laptop (MFD 2024/05)
Operating System: Windows 11
Processor: Intel(R) Core(TM) Ultra 7 155H @ 3.80 GHz
RAM: 32.0 GB (31.5 GB usable)
Graphics and NPU VRAM: 16.0 GB usable (Intel(R) Arc(™) Graphics and Intel(R) AI Boost)
System Type: 64-bit operating system, x64-based processor
```
